nncf/openvino/graph/nncf_graph_builder.py

```diff
@@ -202,11 +210,21 @@ class OVConstantLayerAttributes(BaseLayerAttributes):
     This class stores mapping weights port indices to constant name and shape.
```

Comment:
What do you think about renaming  `OVConstantLayerAttributes` -> `OVLayerAttributes`?

Reply:
Seems good. 
But in this case wouldn't we need to create `OVLayerAttributes` for each layer, regardless of its metatype?